src/include/daos_cont.h

```diff
@@ -138,6 +138,35 @@ int
 daos_cont_create(daos_handle_t poh, const uuid_t uuid, daos_prop_t *cont_prop,
 		 daos_event_t *ev);
 
+/**
+ * Create a new container with label \a label on the storage pool connected
+ * by \a poh.
+ *
+ * \param[in]	poh	Pool connection handle.
+ * \param[in]	label	Required, label property of the new container.
+ *			Supersedes any label specified in \a cont_prop.
+ * \param[in]	cont_prop
+ *			Optional, container properties pointer
+ *			that if specified must not include an entry
+ *			with type DAOS_PROP_CO_LABEL.
+ * \param[out]	uuid	Optional, pointer to uuid_t to hold the
```

Comment:
alignment

Reply:
It seems to render OK when looking at it in a terminal (e.g., with "less daos_cont.h") and seems to look reasonably aligned here. https://github.com/daos-stack/daos/blob/kccain/daos_7174_create_destroy_bylabel/src/include/daos_cont.h  Is it maybe only the pull request rendering making the uuid argument misaligned?